lib/messagebird/client.rb

```diff
@@ -235,6 +237,25 @@ def voice_message_create(recipients, body, params={})
         params.merge({ :recipients => recipients, :body => body.to_s })))
     end
 
+    def call_create(source, destination, call_flow, params={})
+      Call.new(request(
+        :post,
+        'calls',
+        params.merge({ :source => source, :destination => destination, :callFlow => call_flow.to_s })))
+    end
+
+    def call_list
```

Comment:
The docs aren't entirely clear on this but judging from the [examples](https://developers.messagebird.com/api/voice-calling#calls), it appears this endpoint supports pagination. Maybe it's good to figure out how they do it and then support it here.

Reply:
done!